src/plugins/explore/public/components/patterns_table/utils/utils.ts

```diff
@@ -62,21 +62,27 @@ export const createSearchPatternQueryWithSlice = (
   patternsField: string,
   usingRegexPatterns: boolean,
   patternString: string,
-  timeField: string,
+  timeField: string | undefined,
   pageSize: number,
   pageOffset: number
 ) => {
   // TODO: switch this logic back to adding onto the createSearchPatternQuery
```

Comment:
Completely optional for this PR but if you would be able to fulfill this TODO and switch the query here, that would be really nice! I believe the associated backend bug that this workaround was required for has been fixed.